src/ActionSheet/index.tsx

```diff
@@ -71,8 +72,13 @@ export default class ActionSheet extends React.Component<Props, State> {
           flex: 1,
         }}>
         {React.Children.only(this.props.children)}
-        {overlay}
-        {isVisible ? this._renderSheet() : null}
+
+        {isVisible && (
+          <Modal animationType="none" transparent={true} onRequestClose={this._selectCancelButton}>
```

Comment:
Let's use a `useModal` prop to determine if the Modal should be used vs the View.

Reply:
Changes addressed... Code may be better, so, if you have more comments, just let me know and I can do them

Reply:
would we want to update the README with the new prop as part of this change as well?

Reply:
@sfuqua I didn't see any documentation about it.
What I'm adding here, is a new prop on `showActionSheetWithOptions`'s params. I believe that the only documented props are the `<ActionSheetProvider>` props.

Well, what I did was actually look for the documentation of `cancelButtonIndex` (one of the `showActionSheetWithOptions` params), and I didn't find it, so, seems like none of them are documented. Only in the TS types file.

Reply:
Oh no... My bad... I didn't read https://github.com/expo/react-native-action-sheet#universal-props

I'm gonna add the doc tonight

Reply:
No worries -- yes lets add that to the Android only section.

Reply:
Done!